jumanji/environments/packing/knapsack/env.py

```diff
@@ -103,7 +103,7 @@ def __init__(
         """
 
         self.generator = generator or RandomGenerator(
-            num_items=20,
+            num_items=50,
```

Comment:
If we are changing num_items for Knapsack, should we also increment the version in the registry?

Reply:
This point is addressed in the PR description. Let me know if this works for you.

Reply:
Ah yes, gotcha.